dashboard/config/blocks/PoemBot/PoemBot_setBackgroundEffect.json

```diff
@@ -27,17 +27,17 @@
             "Dark Circles",
             "\"darkCircles\""
           ],
-          [
-            "Cloudy",
```

Comment:
i'm not seeing this in the code anywhere, but just double-checking -- do we need to clean up any code now that we're removing the "cloudy" effect?

Reply:
Nope, it wasn't used in code anywhere yet.